workflows/ZWEWRF0n.xml

```diff
@@ -302,4 +302,82 @@ sh ./zowe-configure-instance.sh -c "${instance-zowe_instance_path}"</inlineTempl
             <!-- zosmfOutput -->
         </template>
     </step>
+    <step name="change_instance_configuration" optional="false">
+        <title>Change instance configuration</title>
+        <description>This step changes the configuration of the newly created instance
+based on the values input by the user.</description>
+        <prereqStep name="define_variables"/>
+        <!--variableValues-->
+        <!-- condition -->
+        <instructions substitution="false">Run this step to change the instance configuration.</instructions>
+        <weight>1</weight>
+        <skills>None</skills>
+        <autoEnable>true</autoEnable>
+        <canMarkAsFailed>false</canMarkAsFailed>
+        <template>
+            <inlineTemplate substitution="true">//REPLFILE EXEC PGM=IEBGENER
+//SYSPRINT DD SYSOUT=*
+//SYSIN    DD DUMMY
+//SYSUT2   DD PATH='${instance-zowe_instance_path}/instance.env',
```

Comment:
By coding the path directly in the DD the length is limited to 36 characters (total line must be 71 or less). 
If we want we can allow a longer path (57 chars) by using a SET statement
// SET INST='${instance-zowe_instance_path}'
// SET QUOTE=''''
...
//SYSUT2   DD PATH=&QUOTE.&INST/instance.env',

Either way, we should document the maximum path length where customers can specify the path. This likely also applies to other paths like zowe_install_path.

Reply:
z/OSMF is a little bit different. The max record length is set to 1024, which means each individual line of the job submitted from that step can be up to 1024 characters. It is a little specialty that z/OSMF has, otherwise, variables would require a lot of additional checks.

Reply:
Didn't know that. Thanks for the education.